modules/sbo-categories-of-exposable-binding-data.adoc

```diff
@@ -23,6 +23,23 @@ metadata:
 ----
 
 
+== Exposing a constant value as the binding item
+The following examples show how to expose constant value as annotations:
+
+.Example: Exposing constant value as annotations
+[source,yaml]
+----
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: hippo
+  namespace: my-postgresql
+  annotations:
+    "service.binding/type": "postgresql" <1>
```

Comment:
I don't think we can directly specify a string value. See the example: https://docs.openshift.com/container-platform/4.9/applications/connecting_applications_to_services/getting-started-with-service-binding.html

Reply:
Yes, we can - this was added as a part of https://github.com/redhat-developer/service-binding-operator/pull/1040

Reply:
I believe there is no change to be done here. @pedjak please confirm.